shared/src/library/kotlin/org/jetbrains/kotlin/konan/library/SearchPathResolver.kt

```diff
@@ -203,7 +203,7 @@ internal fun SearchPathResolverWithTarget.libraryMatch(candidate: KonanLibraryIm
     val candidateAbiVersion = candidate.versions.abiVersion
     val candidateLibraryVersion = candidate.versions.libraryVersion
 
-    if (resolverTarget != null && !candidate.targetList.contains(resolverTarget.visibleName)) {
```

Comment:
Why this change?

Reply:
Found that it's always true when was fixing failing test.